src/editor-sidebar/update-panel.js

```diff
@@ -210,9 +210,12 @@ export const UpdateThemePanel = () => {
 						</>
 					}
 					// eslint-disable-next-line @wordpress/i18n-no-collapsible-whitespace
-					placeholder={ __( `Plugin Name
+					placeholder={ __(
+						`Plugin Name
 https://wordpress.org/plugins/plugin-name/
-Plugin Description`, 'create-block-theme' ) }
+Plugin Description`,
+						'create-block-theme'
+					) }
 					value={ theme.recommended_plugins }
```

Comment:
It seems like the translation domain (`create-block-theme`) is already there.

Reply:
https://github.com/WordPress/create-block-theme/pull/499/commits/bdd7389efcc9b631c9dc32f3cf849907374ac1d4

I added it here, but it gave me a JS syntax error, so I adjusted the whitespace separately.

Reply:
Yeah, looks like the text domain was missing from here.